pkg/controller/endpointslice/topologycache/topologycache.go

```diff
@@ -254,3 +263,12 @@ func (t *TopologyCache) getAllocations(numEndpoints int) map[string]Allocation {
 
 	return allocations
 }
+
+func hasExcludedLabels(labels map[string]string) bool {
+	for key := range labels {
+		if excludedLabels.Has(key) {
+			return true
+		}
+	}
+	return false
```

Comment:
we can save a global variable if we embed the excluded labels in the functions, the labels parameters is also a map, so we don't have to iterate
```
func hasExcludedLabels(labels map[string]string) bool {
    if len(labels) == 0 {
           return false
   }
   if _, ok := labels["node-role.kubernetes.io/control-plane"]; ok {
            return true
   }
   if _, ok := labels["node-role.kubernetes.io/master"]; ok {
            return true
   }
   return false
}

Reply:
Updated, thanks!